app/api/orders.py

```diff
@@ -311,7 +311,8 @@ def before_update_object(self, order, data, view_kwargs):
         :param view_kwargs:
         :return:
         """
-        check_billing_info(data)
+        if data.get('amount') or data.get('is_billing_enabled'):
```

Comment:
Why `or`, shouldn't it be `and`?

Reply:
It should be `or` only @kushthedude
`and` doesn't make sense here